src/client/components/Profile.js

```diff
@@ -31,7 +31,15 @@ const Profile = () => (
               <EducationPanel />
             </LeftColumn>
             <RightColumn>
+              {/*
+                For future reference: this will be re-enabled
+                when it is possible to confirm membership in external DAOs.
+
+                What is left in place: modal allowing to add membership record,
```

Comment:
By "What is left in place", do you mean "What works now"?

Reply:
Not quite working, more like... anabiosis. The code is there, but it is not accessible without Org Panel and modals.